### PR TITLE
[test](Nereids) add grouping sets test

### PR DESCRIPTION
**Pull request by feiniaofeiafei**
add grouping sets regression test

### CODE REVIEW

**Comment by morrySnow** [comment]
run buildall

File: regression-test/suites/nereids_rules_p0/grouping_sets/test_grouping_sets_combination.groovy [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        "is_being_synced" = "false",
+        "storage_format" = "V2",
+        "light_schema_change" = "true",
+        "disable_auto_compaction" = "false",
+        "enable_single_replica_compaction" = "false"
+        );
+    """
+
+    sql """insert into agg_test_table_t(`k1`,`k2`,`k3`) values('20231026221524','PA','adigu1bububud');"""
+    qt_select1 """
```
Comment (morrySnow) [inline regression-test/suites/nereids_rules_p0/grouping_sets/test_grouping_sets_combination.groovy:99]: not use numeric suffix. use meaningful names

File: regression-test/suites/nereids_rules_p0/grouping_sets/test_grouping_sets_combination.groovy [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        "replication_allocation" = "tag.location.default: 1",
+        "is_being_synced" = "false",
+        "storage_format" = "V2",
+        "light_schema_change" = "true",
+        "disable_auto_compaction" = "false",
+        "enable_single_replica_compaction" = "false"
+        );
+    """
+
+    sql """insert into agg_test_table_t(`k1`,`k2`,`k3`) values('20231026221524','PA','adigu1bububud');"""
```
Comment (morrySnow) [inline regression-test/suites/nereids_rules_p0/grouping_sets/test_grouping_sets_combination.groovy:98]: add
```
sql """
sync
"""
```

after insert for stable result

**Comment by feiniaofeiafei** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**